mesheryctl/internal/cli/root/system/reset_test.go

```diff
@@ -0,0 +1,88 @@
+package system
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+//var update1 = flag.Bool("update", true, "update golden files")
+
+// This is an Integration test
+func TestResetCmdIntegration(t *testing.T) {
+	// skipping this integration test with --short flag
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	SetupContextEnv(t)
+	tests := []utils.CmdTestInput{
+		{
+			Name:             "Reset the meshery config file with docker platform",
+			Args:             []string{"reset", "-y"},
+			ExpectedResponse: "reset.output.golden",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.Name, func(t *testing.T) {
+			// setting up log to grab logs
+			var buf bytes.Buffer
+			log.SetOutput(&buf)
+			utils.SetupLogrusFormatter()
```

Comment:
We already have a function for this in testing.go.
`SetUpLogrusGrabTesting` , you can use that instead.

Reply:
Yeah sure @Revolyssup